### PR TITLE
Support menu's nocheckmark attribute

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #88 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

Added support for auro-menu's `nocheckmark` attribute, while making checkmark in select the default.

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: docs/partials/demo.md [hunk 1]
```diff
@@ -43,6 +43,19 @@ A baseline `<auro-select>` using `<auro-menu>` and `<auro-menuoption>` elements.
 
 </auro-accordion>
 
+### Example with no checkmark
```
Comment (blackfalcon) [inline docs/partials/demo.md:46]: Why is the following example missing? 

```
### Example with nested menus

<div class="exampleWrapper">
  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/withSubmenus.html) -->
  <!-- AURO-GENERATED-CONTENT:END -->
</div>

<auro-accordion lowProfile justifyRight>
  <span slot="trigger">See code</span>

<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/withSubmenus.html) -->
<!-- AURO-GENERATED-CONTENT:END -->

</auro-accordion>
```

File: docs/partials/demo.md [hunk 2]
```diff
@@ -43,6 +43,19 @@ A baseline `<auro-select>` using `<auro-menu>` and `<auro-menuoption>` elements.
...
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/noCheckmark.html) -->
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
+
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/noCheckmark.html) -->
+<!-- AURO-GENERATED-CONTENT:END -->
+
```
Comment (blackfalcon) [inline docs/partials/demo.md:58]: Missed closing `</auro-accordion>`. The demo page UI breaks when rendered locally.

File: src/auro-select.js [hunk 3]
```diff
@@ -26,6 +26,7 @@ import styleCssFixed from './style-fixed-css.js';
  * @prop {String} value - Value selected for the component.
  * @prop {Boolean} error - When attribute is present element shows error state.
  * @prop {Boolean} disabled - When attribute is present element shows disabled state.
+ * @prop {Boolean} checkmark - When true, selected option will show the checkmark.
```
Comment (blackfalcon) [inline src/auro-select.js:29]: This is incorrect. The default should be with the checkmark and the new feature is to remove the checkmark. Update this feature to be `noCheckmark`.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

**Comment by jordanjones243** [comment]
New PR created after merge conflict mess: https://github.com/AlaskaAirlines/auro-select/pull/103

**Pull request closed without merging**